python/ray/air/config.py

```diff
@@ -16,9 +16,6 @@
 from ray.util.annotations import PublicAPI
 from ray.widgets import Template, make_table_html_repr
 
-
-# Move here later when ml_utils is deprecated. Doing it now causes a circular import.
```

Comment:
Not sure what this was originally referring to, but this is just a loose comment now...

Reply:
Ah this was about SyncConfig I believe. I think we can leave it as is for now and re-factor with the execution package